api-client/src/main/java/com/xing/api/data/SafeCalendar.java

```diff
@@ -101,6 +101,12 @@ public String toString() {
               + ']';
     }
 
+    @Override
+    public void set(int field, int value) {
+        super.set(field, value);
+        if (field == DAY_OF_YEAR) super.complete();
```

Comment:
I feel like this isn't the last time we hack this class. I hope that #43 will help with all these problems...